consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/ConsistencyChecker.java

```diff
@@ -203,37 +204,22 @@ public void run() {
         List<MigrationJob> jobs = new ArrayList<>();
         try {
             DataFindRequest findRequest = new DataFindRequest("migrationJob", migrationJobEntityVersion);
-            /*
-                and:
-                    whenAvailableDate <= new Date()
-                    or:
-                        jobExecutions# = 0
-                        not:
-                            jobExecutions.jobStatus $in ['COMPLETED_SUCCESS', 'COMPLETED_PARTIAL']
-                        and:
-                            not:
-                                jobExecutions.jobStatus $not_in ['COMPLETED_SUCCESS', 'COMPLETED_PARTIAL']
-                            jobExecutions.actualStartDate < (new Date() - expectedExecutionMilliseconds)
-
-             */
+
             findRequest.where(
-                and(
-                    withValue("whenAvailableDate <= " + ClientConstants.getDateFormat().format(new Date())),
-                    or(
-                        not(
-
-                            withSubfield("jobExecutions", withValue("jobStatus $in [COMPLETED_SUCCESS, COMPLETED_PARTIAL]"))
-                        ),
-                        and(
-                            not(
-
-                                    withSubfield("jobExecutions", withValue("jobStatus $not_in [COMPLETED_SUCCESS, COMPLETED_PARTIAL]"))
-                            ),
-                            // TODO check how lightblue will handle the next comparison (if it can handle this case), suggestions?
-                            withValue("whenAvailableDate + expectedExecutionMilliseconds < " + ClientConstants.getDateFormat().format(new Date()))
-                        )
+                    and(
+                            withValue("configurationName = " + configuration.getConfigurationName()),
+                            withValue("whenAvailableDate <= " + ClientConstants.getDateFormat().format(new Date())),
+                            or(
+                                    not(
+                                            withSubfield("jobExecutions", withValue("jobStatus $in [COMPLETED_SUCCESS, COMPLETED_PARTIAL]"))
+                                    ),
+                                    and(
+                                            not(
+                                                    withSubfield("jobExecutions", withValue("jobStatus $not_in [COMPLETED_SUCCESS, COMPLETED_PARTIAL]"))
```

Comment:
Made sense when I wrote that originally I guess when discussing with @luan-cestari 
Looking at it now it doesn't.  Here's the original post about it: https://github.com/lightblue-platform/lightblue-migrator/pull/120#discussion_r26861317

This is where having #142 would help.. could remove any ambiguity by testing the code.